qemu/include/qemu/log.h

```diff
@@ -30,20 +30,207 @@
 
 /* Logging functions: */
 
-/* log only if a bit is set on the current loglevel mask:
- * @mask: bit to check in the mask
- * @fmt: printf-style format string
- * @args: optional arguments for format string
+/* To verbose logging, enable the next line. */
+//#define LOGGING_ENABLED // to enable logging
```

Comment:
We can add a cmake variable for this. If you are not familiar, I can do this for you.